templates/mks/member_embed.html

```diff
@@ -0,0 +1,57 @@
+<html lang="he">
+  <head>
+    <meta http-equiv="Content-Type" language="he" content="text/html;charset=utf-8">
+    <link rel="stylesheet" href="/static/css/mk-iframe.css" type="text/css" />
+  </head>
+<body>
+  <div class="oknesset_frame" id="oknesset_mk_{{ object.id }}">
+    <div class="oknesset_content_top">
+      {% url 'member-detail' id as mk_url %}
+      <a target="_blank" href="{{ mk_url }}">
+        <img class="oknesset_image" src="{{ object.img_url }}">
+      </a>
+      <div class="oknesset_name">
+        <a target="_blank" href="{{ mk_url }}">{{ object.name }}</a>
+        {% if is_current %}- ח"כ מכהן{% endif %}
+      </div>
+      <div class="oknesset_roles">
+      </div>
+      <div class="oknesset_law_flow">
+        <a target="_blank" href="http://127.0.0.1:8000/bill/?member={{ object.id }}&stage=proposed" class="oknesset_proposals">
```

Comment:
Two problems with this line and the one at 24:

- The URL is hard linked to localhost:8000.
- `&` Should be `&amp;`.